packages/client/webapp/src/components/ProjectCard.vue

```diff
@@ -0,0 +1,134 @@
+<script setup lang="ts">
+// import Button from '@/components/Button.vue';
+import IconList24 from '@carbon/icons-vue/es/list/24';
+import IconAddFilled32 from '@carbon/icons-vue/es/add--filled/32';
+
+interface Props {
+	category: string;
+	projectName: string;
+	isNewProjectCard: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+	category: '',
+	projectName: '',
+	isNewProjectCard: false
+});
+
+function createNewProject() {}
+</script>
+
+<template>
+	<div v-if="!props.isNewProjectCard" class="project-card">
+		<div class="details">
+			<div class="name">{{ props?.projectName }} {{ props?.p }}</div>
+			<div class="actions">
+				<IconList24 />
+			</div>
+		</div>
+	</div>
+	<div v-else class="new-project-card">
+		<IconAddFilled32 class="icon-add-filled" @onclick="createNewProject" />
+		<div class="details">
+			<div class="create-new-project">Create New Project</div>
+		</div>
+	</div>
+	<!-- <div v-else class="new-project-card">
+        <Button @onclick="createNewProject">Create New Project
+            <IconAddFilled32 />
+        </Button>
+    </div> -->
+</template>
+
+<style scoped>
+.project-card,
+.new-project-card {
+	color: var(--un-color-black-80);
+	border: 1px solid var(--un-color-black-20);
+	background-color: var(--un-color-black-5);
+	font-weight: 500;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+	height: 15rem;
+	min-width: 20rem;
+	border-radius: 0.5rem;
+	margin: 0.5rem;
+	transition: 0.2s;
+	text-align: left;
+}
+
+.project-card:hover {
+	height: 20rem;
+	min-width: 25rem;
+	cursor: pointer;
+	/* position: fixed; */
+}
```

Comment:
If you are changing the size of the card as an hover state, then you should remove the overflow on the carousel to avoid it being cut, and you should not have to change the dimensions of the carousel. See video on my main PR comment.

Reply:
When I remove `overflow-y: hidden` I get this: 
<img width="613" alt="image" src="https://user-images.githubusercontent.com/28237258/196784804-08d13b72-ad52-4525-9c98-952860f02315.png">

Is changing the size important to implement at this stage? Expanding the height of the footer to show actions could have a similar effect/function.